assets/js/modules/analytics-4/components/setup/SetupEnhancedMeasurementSwitch.js

```diff
@@ -66,6 +66,10 @@ export default function SetupEnhancedMeasurementSwitch() {
 		} )
 	);
 
+	const autoSubmit = useSelect( ( select ) =>
+		select( CORE_FORMS ).getValue( FORM_SETUP, 'autoSubmit' )
+	);
```

Comment:
If needed, this could be a case to use `getValue` in the `useMount` since we're only interested in the initial value once.